static/js/publisher/market/state.js

```diff
@@ -7,9 +10,18 @@ const allowedKeys = [
   'contact',
   'private',
   'public_metrics_enabled',
-  'public_metrics_blacklist'
+  'public_metrics_blacklist',
```

Comment:
If this file starts to be reused in different places I feel it may require some refactor.

It was developed for market page form and has a hardcoded list of market page inputs. Now we added other inputs here that we moved to a different page. So now we have a 'generic' form state library that contains list of fields from 2 different forms.

Maybe `state` could stay generic but it would receive `allowedKeys` and `transform` functions as parameters when created/initialized? This way shared functionality would stay here while form dependant list of the fields would be separated.

Reply:
See above re. refactor